index.js

```diff
@@ -230,6 +282,9 @@ module.exports = {
                             filename = titleSlug(testTitle);
                         }
 
+                        if (autoCaptureOptions.indexOf('source') !== -1) {
```

Comment:
this call should be added to the existing promise chain to make sure its errors are handled in the existing catchers.

```js
let sourcePromise = (autoCaptureOptions.indexOf('source') !== -1) ? nemo.screenshot.source(filename) : Promise.resolve(true);
return sourcePromise
    .then(nemo.screenshot.snap(filename))
    .then(oclick.apply(this, arguments));
```

I think that should work.